packages/chart/components/Chart.tsx

```diff
@@ -0,0 +1,39 @@
+`;
+export interface ChartProps {
+  data: {
+    [label: string]: number;
+  };
+  type: string;
+}
+
+class Chart extends React.PureComponent<ChartProps, {}> {
+  public static defaultProps = {
+    type: "Line"
+  };
+  public render() {
+    const { data, type } = this.props;
+    const graphData = {
+      labels: Object.keys(data),
+      series: [Object.values(data)]
+    };
+    return (
+      <ChartistGraph
+        type={type}
+        data={graphData}
+        options={{
+          // @ts-ignore
+          fullWidth: true
```

Comment:
Why is this needed?

Reply:
Because fullWidth is not part of the interface in the options of the library we are using.